lib/puppet/type/incron_system_table.rb

```diff
@@ -66,7 +70,28 @@
     desc 'The incron "masks" to apply'
 
     munge do |value|
-      Array(value).join(',')
+      value = Array(value).join(',').split(',')
+
```

Comment:
Wouldn't it be better to strip the unsupported features from valid_incron_masks, instead, so that invalid masks were not silently dropped?  As a user, I would be confused why my masks passed validation but didn't appear in the incron table.

Reply:
@lnemsick-simp I don't know if that fact is available at that scope. I think that this is a reasonable improvement should probably be an additional ticket due to the importance of the fix.